src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -247,7 +247,11 @@ public function create_item( $request ) {
 			return $autosave_id;
 		}
 
-		$autosave = get_post( $autosave_id );
+		$autosave = $this->get_post( $autosave_id );
```

Comment:
```suggestion
		$autosave = $this->parent_controller->get_post( $autosave_id );
```
Could we not just do this?

Reply:
I've replied here: https://github.com/WordPress/wordpress-develop/pull/7076#discussion_r1795450195